bin/configure-tower-projects.py

```diff
@@ -365,37 +382,54 @@ def create(self) -> dict:
         response = self.tower.request("POST", endpoint, json=data)
         return response["workspace"]
 
-    def add_participant(self, user: str, role: str) -> int:
+    def add_participant(self, role: str, user: str = None, team_id: int = None) -> int:
         """Add user to the workspace (if need be) and return participant ID
 
         Args:
-            user (str): Email address for the user
             role (str): 'owner', 'admin', 'maintain', 'launch', or 'view'
+            user (str): Email address for the user
+            team_id (int): Team identifier
 
         Returns:
             int: Participant ID for the user in the given workspace
         """
         # Attempt to add the user as a participant of the given workspace
         endpoint = f"/orgs/{self.org.id}/workspaces/{self.id}/participants"
-        member_id = self.org.members[user]["memberId"]
-        data = {
-            "memberId": member_id,
-            "teamId": None,
-            "userNameOrEmail": None,
-        }
+        if user and not team_id:
+            member_id = self.org.members[user]["memberId"]
+            identifier = member_id
+            data = {
+                "memberId": member_id,
+                "teamId": None,
+                "userNameOrEmail": None,
+            }
+        elif not user and team_id:
+            identifier = team_id
+            data = {
+                "memberId": None,
+                "teamId": team_id,
+                "userNameOrEmail": None,
+            }
+        else:
+            raise ValueError(
+                "Must provide value for exactly one of `user` or `team_id`."
```

Comment:
nit: Why not user_id when its team_id?

Reply:
If something were to be `user_id`, it would be the `member_id`. Currently, `user` contains emails because that's what's stored in the Sceptre config files. I'm not using `email` as the variable name because it can technically contain a username (as indicated by the Tower field, `userNameOrEmail`). I'm not particularly eager to use `user_name_or_email` either. 

Can you think of a better variable name?

Reply:
I could refactor the code so that member IDs are tracked as soon as they're obtained instead of relying on `self.org.members` to retrieve the member ID in this function.

Reply:
I see. This makes sense. No need to refactor, user makes sense here.